integration-tests/it-consumer/src/main/java/org/apache/servicecomb/it/deploy/MicroserviceDeploy.java

```diff
@@ -43,7 +43,12 @@ public MicroserviceDeployDefinition getMicroserviceDeployDefinition() {
   @Override
   protected String[] createCmds() {
     // must set jar at the end of the cmds
-    return new String[] {"java", "-jar"};
+    return new String[] {"java", "-jar",
+        //add a property definition to enable quick search in taskmgr
+        "-DCombDeploy=true",
```

Comment:
can we use jps to find the process?

Reply:
It's just a marker to search in windows task manager, I'll remove it later.